share/ext/luajit/build_luajit.sh

```diff
@@ -150,10 +150,32 @@ case $1 in
 		export TARGET_SYS=iOS
 		;;
 	armv7-android)
+		# using insttructions from here: file:///Users/mawe/work/defold-ps4/share/ext/luajit/tmp/doc/install.html
```

Comment:
I had some issues using our"build system" now for Android, so I added the recommended settings from the official page instead which worked fine.